package_for_correction.py

```diff
@@ -1,10 +1,11 @@
 #!/usr/bin/env python
 #
-# Script to package data for manual correction from SpineGeneric adapted for ukbiobank cord CSA project.
+# Script to package data for manual correction from SpineGeneric adapted for canproco project.
```

Comment:
We should maybe remove that it is adapted for a certain project since the point of this is for it to be generalizable!

Reply:
Great catch! You are right. I missed this. Thank you!
Fixed in https://github.com/spinalcordtoolbox/manual-correction/pull/2/commits/bb3dee46ee5d0419fa4e71b46a2355b9eb67248e